app/scripts/views/footer/footer-partial.html

```diff
@@ -14,11 +14,11 @@
                 </div>
                 <div class="col-xs-3">
                     <a ui-sref="about"><h3>About</h3></a>
-                    <p>MuseumStat is a powerful online resource to support museums, communities, and researchers in gaining useful insights into the scope and reach of museums and the communities they serve.</p>
+                    <p>MuseumStat is a powerful online resource to support museums, communities, and researchers in gaining useful insights into the scope and reach of museums and the communities they serve.</p><a ui-sref="about">Learn more</a>
                 </div>
                 <div class="col-xs-3">
                     <a ui-sref="contact"><h3>Contact</h3></a>
-                    <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt. Duis aute irure dolor in reprehenderit in voluptate fugiat esse cillum dolore eu fugiat nulla pariatur fugiat reprehenderit. </p>
+                    <p>We want to ensure that MuseumStat meets your needs and provides you with valuable information. Learn more and let us know how we can help.</p><a ui-sref="contact">Learn more</a>
```

Comment:
I think the modifications on this page need to be added to `404.html` as well, so that #90 can be closed.